app/views/shared/_no_evaluation_form_alert.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="grid-row margin-top-105">
+  <div class="grid-col-12 tablet:grid-col-6">
+    <div class="usa-alert usa-alert--error padding-top-105" role="alert">
+      <div class="usa-alert__body">
+        <h4 class="usa-alert__heading"><%= t('alerts.no_evaluation_form.heading') %></h4>
+        <p class="usa-alert__text">
+          <%= t('alerts.no_evaluation_form.text') %>
+        </p>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
having seen #260, I think this could be a reusable template component pretty easily for reuse throughout the app:

```suggestion
<div class="grid-row margin-top-105">
  <div class="grid-col-12 tablet:grid-col-6">
    <div class="usa-alert usa-alert--error padding-top-105" role="alert">
      <div class="usa-alert__body">
        <h4 class="usa-alert__heading"><%= alert_heading %></h4>
        <p class="usa-alert__text">
          <%= alert_text %>
        </p>
      </div>
    </div>
  </div>
</div>
```